scripts/run.py

```diff
@@ -525,7 +521,7 @@ def capacity_subscriber(const_cap, subscribers):
     cap_sub : float
         Capacity per subscriber
     """
-    cap_sub = const_cap * 0.5 * subscribers
+    cap_sub = const_cap * 0.5 / subscribers
```

Comment:
What is 0.5? This should be in the parameters dictionary with a proper key. Please make sure all parameters are in the parameters dictionary.